lib/Doctrine/ORM/Query/Expr/Between.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Doctrine\ORM\Query\Expr;
+
+class Between
+{
+    const NOT_BETWEEN = 'NOT BETWEEN';
+
+    const BETWEEN = 'BETWEEN';
+
+    /**
+     * @var string
+     */
+    protected $operator;
+
+    /**
+     * @var mixed
+     */
+    protected $key;
+
+    /**
+     * @var int|string
+     */
+    protected $min;
+
+    /**
+     * @var int|string
+     */
+    protected $max;
+
+    /**
+     * Constructor.
```

Comment:
This line doesn't say anything we should either remove it or have meaningful explanation (I'd simply remove it 😛)